vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -35,34 +35,112 @@ def __init__(self, model_xml: bytes,
 
         self.ie = ie_core or IECore()
 
-        # Find the optimal number of InferRequests for this device
-        supported_metrics = self.ie.get_metric(
-            device_name, _SUPPORTED_METRICS)
-        if _RANGE_FOR_ASYNC_INFER_REQUESTS in supported_metrics:
-            low, high, _ = self.ie.get_metric(
-                device_name, _RANGE_FOR_ASYNC_INFER_REQUESTS)
-            # Cap the n_requests, because sometimes high_n crashes the system
-            # TODO(Alex): Figure out _why_ hddl crashes when set to 'high'
-            n_requests = max(0, min(low * 2, high))
-        else:
-            # Use the devices default
-            n_requests = 0
+        # If the device is a MULTI device, use the default num_requests
+        is_multi_device = device_name.lower().startswith("multi")
+
+        num_requests = 0
+        if not is_multi_device:
+            # Find the optimal number of InferRequests for this device
+            supported_metrics = self.ie.get_metric(
+                device_name, _SUPPORTED_METRICS)
+            if _RANGE_FOR_ASYNC_INFER_REQUESTS in supported_metrics:
+                low, high, _ = self.ie.get_metric(
+                    device_name, _RANGE_FOR_ASYNC_INFER_REQUESTS)
+                # Cap the num_requests, because setting it too high can crash
+                # TODO(Alex): Figure out _why_ hddl crashes when set to 'high'
+                num_requests = max(0, min(low * 2, high))
 
         self.net: IENetwork = self.ie.read_network(
             model=model_xml,
             weights=weights_bin,
             init_from_buffer=True)
 
-        self.exec_net: ExecutableNetwork = self.ie.load_network(
-            network=self.net,
-            device_name=device_name,
-            num_requests=n_requests)
+        try:
```

Comment:
The purpose of this change is to load the network and catch RuntimeErrors which are the main source of crashes when a misconfigured HDDL environment is found. 

Rarely, an environment might be configured fine, but HDDL is in a weird state and not working. 

In these cases, we put a critical log out, and attempt reloading the network on "CPU" only.